vnext/src/Libraries/Components/Pressable/Pressable.windows.js

```diff
@@ -179,6 +190,8 @@ function Pressable(props: Props, forwardedRef): React.Node {
     // [Windows
     onBlur,
     onFocus,
+    onKeyDown,
+    onKeyUp,
```

Comment:
Shouldn't we be passing along the `keyDownEvents` and `keyUpEvents` props?

Reply:
This is currently broken, will fix i follow-up PRs. 
Filed: 
https://github.com/microsoft/react-native-windows/issues/7501
https://github.com/microsoft/react-native-windows/issues/7500